manager/src/main/java/com/redhat/service/smartevents/manager/ProcessorServiceImpl.java

```diff
@@ -179,12 +179,16 @@ private Processor doCreateProcessor(Bridge bridge, String customerId, String own
     }
 
     private Action resolveAction(Action action, String customerId, String bridgeId, String processorId) {
+        if(action == null)
```

Comment:
please add curly braces for `if` clauses

Reply:
Ok. Added.